tests/integrations/raftstore/test_scale_pool.rs

```diff
@@ -0,0 +1,135 @@
+// Copyright 2021 TiKV Project Authors. Licensed under Apache-2.0.
+
+use engine_traits::CF_DEFAULT;
+use kvproto::raft_cmdpb::RaftCmdResponse;
+use libc::{getpid, pid_t};
+use procinfo::pid;
+use raftstore::Result;
+use std::collections::HashMap;
+use std::time::Duration;
+use test_raftstore::*;
+use tikv_util::{metrics::get_thread_ids, HandyRwLock};
+
+fn put_with_timeout<T: Simulator>(
+    cluster: &mut Cluster<T>,
+    key: &[u8],
+    value: &[u8],
+    timeout: Duration,
+) -> Result<RaftCmdResponse> {
+    let mut region = cluster.get_region(key);
+    let region_id = region.get_id();
+    let req = new_request(
+        region_id,
+        region.take_region_epoch(),
+        vec![new_put_cf_cmd(CF_DEFAULT, key, value)],
+        false,
+    );
+    cluster.call_command_on_node(0, req, timeout)
+}
+
+#[test]
+fn test_increase_pool() {
+    let mut cluster = new_node_cluster(0, 1);
+    cluster.cfg.raft_store.store_batch_system.pool_size = 1;
+    cluster.cfg.raft_store.apply_batch_system.pool_size = 1;
+    cluster.pd_client.disable_default_operator();
+    let fp1 = "poll";
+    // When starting,according to the above parameters, raftstore will
+    // start three Poller threads: apply-0, apply-low-0, rafstore-1-0.
+    // Setting this failpoint can make the current Pollers unable to
+    // process the request. So the first put request will time out.
+    // After expanding the pool size, we have new Poller threads that
+    // are not affected by failpoints, so subsequent put requests will
+    // succeed.
```

Comment:
It is more preferred to add short comments close the code that reflects ours test logic sketch, see `test_request_in_joint_state` for example, also ditto for `test_decrease_pool`

Reply:
Fixed, thanks!